assignments/lambda-classes.js

```diff
@@ -1 +1,115 @@
 // CODE here for your Lambda Classes
+class Person {
+    constructor(attributes) {
+        this.name = attributes.name;
+        this.age = attributes.age;
+        this.location = attributes.location;
+        this.gender = attributes.gender;
+
+    }
+    speak() {
+        //console.log(`Object was removed from the game.`);
+        return `Hello my name is Fred, I am from Bedrock`;
+    }
+}
+
+
+class Instructor extends Person {
+    constructor(InstrucAttributes) {
+        super(InstrucAttributes);
+        this.specialty = InstrucAttributes.specialty;
+        this.favLanguage = InstrucAttributes.favLanguage;
+        this.catchPhrase = InstrucAttributes.catchPhrase;
+    }
+    demo(subject) {
+        return `Today we are learning about ${subject}`;
+    }
+
+    grade(student, subject) {
+        `${student.name} receives a perfect score on ${subject}`;
```

Comment:
Good job passing the student object and getting the name. A lot of people missed this and were just passing student.name as a string.